src/tubeValidation.js

```diff
@@ -434,3 +437,17 @@ export const getTubeList = () => {
 export const getConceptList = () => {
   return tubes.map((tube) => tube.concept);
 };
+
+const getUniqueDeviationReasonsList = () => {
+    const sumDeviationCollections = [...deviationCollection1, ...deviationCollection2, ...deviationCollection3, ...deviationCollection4];
+    const seenConcepts = new Set();
+    
+    return sumDeviationCollections.filter( deviationCollection => {
+        if(seenConcepts.has(deviationCollection.concept)) return false;
+        else {
+            seenConcepts.add(deviationCollection.concept)
+            return true;
+        }
+    })
```

Comment:
```suggestion
    const concepts = new Set([...deviationCollection1, ...deviationCollection2, ...deviationCollection3, ...deviationCollection4]);
    return Array.from(concepts);
```
or something similar.